src/codeflare_sdk/common/kueue/kueue.py

```diff
@@ -19,8 +19,21 @@
 from kubernetes.client.exceptions import ApiException
 
 
-def get_default_kueue_name(namespace: str):
-    # If the local queue is set, use it. Otherwise, try to use the default queue.
+def get_default_kueue_name(namespace: str) -> Optional[str]:
+    """
+    Retrieves the default Kueue name from the provided namespace.
+
+    This function attempts to fetch the local queues in the given namespace and checks if any of them is annotated
+    as the default queue. If found, the name of the default queue is returned.
```

Comment:
add a small comment on the actual label that marks the LQ which marks the queue as the default queue here `kueue.x-k8s.io/default-queue`